app/ab_test/a/templates/Donation_Confirmation_Phone.html.twig

```diff
@@ -116,7 +116,7 @@
 						</div>
 
 						<div class="payment f-left">
-							{% if donation.paymentType == 'PPL' or donation.paymentType == 'MCP' %}
+							{% if donation.paymentType == 'PPL' or donation.paymentType == 'MCP' or donation.paymentType == 'SUB' %}
```

Comment:
This depends on whether we want the status to be reflected here. It may be `status-pledge`, `status-unconfirmed` or `status-booked`.